watertap/unit_models/zero_order/pump_electricity_zo.py

```diff
@@ -90,3 +90,70 @@ def applied_pressure_constraint(b, t):
 
         self._perf_var_dict["Electricity (kW)"] = self.electricity
         self._perf_var_dict["Applied Pressure (bar)"] = self.applied_pressure
+
+
+@declare_process_block_class("PumpVariableEfficiencyZO")
+class PumpVariableEfficiencyZOData(PumpElectricityZOData):
```

Comment:
Although this can work too, I think it would be better, at least for automated documentation reasons, to make this a separate file.

Reply:
That means the yaml should also be separate too.